utility/module.go

```diff
@@ -22,6 +23,9 @@ type utilityModule struct {
 
 	logger  *modules.Logger
 	mempool mempool.TXMempool
+
+	// TODO: initialize
```

Comment:
See #710 by @0xBigBoss  who is working on this.

@0xBigBoss do you think we could merge in your configs work (even though it's not fully done) to unblock soon to come work?